pkgvars

```diff
@@ -0,0 +1,6 @@
+pkgname=pgautofailover
+hubproj=citus-ha
+pkgdesc=''
```

Comment:
Missing description

Reply:
Postgres extension for automated failover and high-availability